packages/plugins/src/cli/setup.ts

```diff
@@ -135,32 +131,9 @@ export const makeCLI = (
           })
           .hide('version'),
       async (argv) => {
-        const { pnpm, foundryup, orgId, sphinxApiKey, alchemyApiKey, owner } =
-          argv
-
-        init(pnpm, foundryup, orgId, sphinxApiKey, alchemyApiKey, owner)
-      }
-    )
-    .command(
-      'remappings',
```

Comment:
We don't really need this anymore because the remapping the user must configure is very simple. So we'll just have them copy paste it from the getting started guide.